.github/workflows/master-branch.yml

```diff
@@ -0,0 +1,43 @@
+name: Release Management
+
+on:
+    push:
+        # branches to consider in the event; optional, defaults to all
+        branches:
+            - master
+
+jobs:
+    build_and_deploy:
+        runs-on: ubuntu-latest
+        steps:
+            -   uses: actions/checkout@v2
+            -   name: Set up JDK 8
+                uses: actions/setup-java@v2
+                with:
+                    java-version: '8'
+                    distribution: 'adopt'
+                    cache: 'maven'
+            # Sonar lint analysis will currently fail du to the java 11 requirement
+            # TODO move sonar lint token into secrets
+            -   name: Execute sonar analysis (will fail until java 11)
```

Comment:
Sonarqube analysis will fail until java 11 is used but i wanted to migrate it to github workflow anyway, therefore i added continue-on-error: true for this step